src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -63,10 +64,13 @@ private static ParticipantType GetParticipantType(bool required)
         private static bool IsValidEmail(string email)
             => new EmailAddressAttribute().IsValid(email);
 
-        public static string GenerateMeetingTitle(Invitation invitation) 
-            => $"Invitation for punch-out, IPO-{invitation.Id}, Project: {invitation.ProjectName}";
+        public static string GenerateMeetingTitle(Invitation invitation, IProjectRepository projectRepository)
+        {
+            var project = projectRepository.GetByIdAsync(invitation.ProjectId).GetAwaiter().GetResult();
+            return $"Invitation for punch-out, IPO-{invitation.Id}, Project: {project.Name}"; 
+        }
 
-        public static string GenerateMeetingDescription(Invitation invitation, string baseUrl, Person organizer)
+        public static string GenerateMeetingDescription(Invitation invitation, string baseUrl, Person organizer, IProjectRepository projectRepository)
```

Comment:
Don't send IProjectRepository as argument into helper. 
Suggestion: Move methods GenerateMeetingTitle + GenerateMeetingDescription into a new static class named MeetingHelper. The only argument into both GenerateMeetingTitle + GenerateMeetingDescription  should be a new helper-class named something as MeetingDto, containing necessary props from Invitation, Person and Project.

Reply:
As agreed, sending in project name instead of project repository instead of creating two new classes.